KimberleeMcLean-Algonquin.md

```diff
@@ -0,0 +1,10 @@
+# Algonquin, IL
+
+### Restaurants
+- Port Edward
```

Comment:
Port Edward is a seafood restaurant that is on the Fox River - there is even a table on a boat that is in the river